static/app/components/editableText.tsx

```diff
@@ -256,9 +193,25 @@ const StyledInput = styled(Input)`
 `;
 
 const InputLabel = styled('div')`
-  width: auto;
-  height: 40px;
-  padding: ${space(1.5)};
-  position: relative;
-  z-index: -1;
+  height: 0;
+  opacity: 0;
+  white-space: pre;
+  padding: 0 ${space(1)};
+`;
+
+const Wrapper = styled('div')<{isDisabled: boolean; isEditing: boolean}>`
+  display: flex;
+
+  ${p =>
+    p.isDisabled
+      ? `
+          ${IconEdit} {
+            display: none;
```

Comment:
if `isDisabled` equals `true`, IconEdit will not be rendered, so I think we can remove this css